src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        try {
+            LocalDate dateFromFormat = getParsedDate(dateFrom);
+            LocalDate dateToFormat = getParsedDate(dateTo);
+            String userInfo = "";
+            StringBuilder strBld = new StringBuilder();//to append result
+            String result = "";
+
+            strBld.append("Report for period ").append(dateFrom).append(" - ").append(dateTo);
+
+            for (int i = 0; i < names.length; i++) {
```

Comment:
You can use foreach here
```suggestion
            for (String name : names) {
```